react-app/src/components/projects/ProjectForms.tsx

```diff
@@ -0,0 +1,176 @@
+      </Grid>
+      <Grid item xs={6}>
+        <AutocompleteFormField
+          name={'TierLevelId'}
+          label={'Assign Tier'}
+          required
+          options={tiers?.map((t) => ({ label: t.Name, value: t.Id })) ?? []}
+        />
+      </Grid>
+      <Grid item xs={12}>
+        <TextFormField fullWidth multiline name={'Description'} label={'Notes'} minRows={3} />
+      </Grid>
+    </Grid>
+  );
+};
+
+export const ProjectFinancialInfoForm = () => {
+  return (
+    <Grid mt={'1rem'} spacing={2} container>
+      <Grid item xs={6}>
+        <TextFormField
+          InputProps={{
+            startAdornment: <InputAdornment position="start">$</InputAdornment>,
+          }}
+          fullWidth
+          numeric
+          name={'Assessed'}
+          label={'Assessed value'}
+          rules={{
+            min: {
+              value: 0.01,
+              message: 'Must be greater than 0.',
+            },
+          }}
+          required
+        />
+      </Grid>
+      <Grid item xs={6}>
+        <TextFormField
+          InputProps={{
+            startAdornment: <InputAdornment position="start">$</InputAdornment>,
+          }}
+          fullWidth
+          numeric
+          name={'NetBook'}
+          label={'Net book value'}
+          rules={{
+            min: {
+              value: 0.01,
+              message: 'Must be greater than 0.',
+            },
+          }}
+          required
+        />
+      </Grid>
+      <Grid item xs={6}>
+        <TextFormField
+          InputProps={{
+            startAdornment: <InputAdornment position="start">$</InputAdornment>,
+          }}
+          numeric
+          fullWidth
+          name={'Estimated'}
+          label={'Estimated market value'}
+          rules={{
+            min: {
+              value: 0.01,
+              message: 'Must be greater than 0.',
+            },
+          }}
+          required
+        />
+      </Grid>
+      <Grid item xs={6}>
+        <TextFormField
+          InputProps={{
+            startAdornment: <InputAdornment position="start">$</InputAdornment>,
+          }}
+          numeric
+          fullWidth
+          name={'Appraised'}
+          label={'Appraised value'}
+        />
+      </Grid>
+      <Grid item xs={6}>
+        <TextFormField
+          InputProps={{
+            startAdornment: <InputAdornment position="start">$</InputAdornment>,
+          }}
+          numeric
+          fullWidth
+          name={'Metadata.salesCost'}
+          label={'Estimated sales cost'}
+        />
+      </Grid>
+      <Grid item xs={6}>
+        <TextFormField
+          InputProps={{
+            startAdornment: <InputAdornment position="start">$</InputAdornment>,
+          }}
+          numeric
+          fullWidth
+          name={'Metadata.programCost'}
+          label={'Estimated program recovery fees'}
+        />
+      </Grid>
+    </Grid>
+  );
+};
+
+export const ProjectDocumentationForm = () => {
```

Comment:
I'm not sure we should be allowing users to change any of these fields.
They weren't editable on old PIMS once the project was submitted. 
Just by being at this phase of the workflow, they must be complete.

Reply:
Yeah, this is sorta placeholder until we merge in the updated tasks api stuff. Maybe instead we present this form after they try changing the projectstatus, since we do need them to choose whether the tasks are completed at some point, but not necessarily at any time on this page.